File: clients/typescript/src/AnonymousSchema_22.ts
import {AnonymousSchema_23} from './AnonymousSchema_23';
export class AnonymousSchema_22 {
  private _request: AnonymousSchema_23;
  private _statusCode: number;
  private _msg: string;

  constructor(input: {
    request: AnonymousSchema_23,
    statusCode: number,
    msg: string,
  }) {
    this._request = input.request;
    this._statusCode = input.statusCode;
    this._msg = input.msg;
  }

  get request(): AnonymousSchema_23 { return this._request; }
  set request(request: AnonymousSchema_23) { this._request = request; }

  get statusCode(): number { return this._statusCode; }
  set statusCode(statusCode: number) { this._statusCode = statusCode; }

  get msg(): string { return this._msg; }
  set msg(msg: string) { this._msg = msg; }
}

File: clients/rust/src/anonymous_schema22.rs
// AnonymousSchema22 represents a AnonymousSchema22 model.
#[derive(Clone, Debug, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
pub struct AnonymousSchema22 {
    #[serde(rename="request")]
    request: Box<crate::AnonymousSchema23>,
    #[serde(rename="status_code")]
    status_code: i32,
    #[serde(rename="msg")]
    msg: String,
}

impl AnonymousSchema22 {
    pub fn new(request: crate::AnonymousSchema23, status_code: i32, msg: String) -> AnonymousSchema22 {
        AnonymousSchema22 {
        Box::new(request),
        status_code,
        msg,
        }
    }
}
